style.css

```diff
@@ -0,0 +1,141 @@
+@import url('http://fonts.cdnfonts.com/css/crete-round');
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+  font-family: 'Roboto', sans-serif;
+}
+
+.toolbar {
+  display: flex;
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  background: #3c3a39;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.toolbar a {
+  width: auto;
+  height: 24px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  text-decoration: none;
+}
+
+.toolbar img {
+  width: 24px;
+  height: 24px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+  background: #1c1a19;
+}
+
+.headline h1 {
+  width: 327px;
+  height: 124px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding-left: 24px;
+  padding-right: 24px;
+  margin-top: -10px;
+}
```

Comment:
- [ ] KIndly make sure to use the same typography styles for the main title in order to meet the provided Figma design. :wink: 

   - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/199078751-a585228a-fa53-4c15-9d54-8e0fc1297c37.png)

   - ### Figma design
   
    ![image](https://user-images.githubusercontent.com/87197412/199078857-0eedaf38-be33-4449-9b20-addf7a140a92.png)